daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/LawlessTraversals.scala

```diff
@@ -8,7 +8,7 @@ import com.daml.scalautil.Statement.discard
 import scala.annotation.tailrec
 import scala.collection.BuildFrom
 
-private[daml] object LawlessTraversals {
+object LawlessTraversals {
```

Comment:
I am not very happy this become fully public.

Reply:
let me fork that inside canton/community/ledger, and add a task to streamline this.
this way we can keep it daml private here :+1: